input/haleu/optimization/once-through/moga_tuning/moga_tuning_input.xml.in

```diff
@@ -0,0 +1,253 @@
+<simulation xml:base="/home/abachmann/transition-scenarios/input/haleu/inputs/" xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
Would it be better to use a relative path here? Or is that not possible?

Reply:
I don't think that's possible. But using the full path here allows for the paths later on to be relative.